### ISSUE TITLE
Proper parameter passing?

### ISSUE THREAD

**Issue by ekkis** [BlackGlory/caddy-proxy#10]
I noticed the prescribed way of passing parameters to the container seems to be via an environment variable, to wit:

`-e CADDY_OPTIONS="--email youremail@example.com`

but in looking at the Caddy documentation, the parameter is actually `-email` with a single dash.  is environment variable above set correctly?

and wouldn't it make sense to allow parameter passing in the more traditional way? e.g.

`$ docker run [...] blackglory/caddy-proxy:0.2.1 -email x@x.com -port 8080
`

**Comment by arnested** [comment]
Correct. Caddys option is with a single dash. It works with double dash as well, though.

It would probably be better to use the single dash as officially documented.

The image used to take patameters in the traditional way. It was actually me that changed it in pull request #3. 

The reason was the move to a base image with a lightway init system as the entrypoint.

I'm not sure it is possible to pass the parameters on from the init system to Caddy. But it would make sense to investigate if something can be done.

**Comment by ekkis** [comment]
ah... that makes sense.  actually, I don't know how to pass parameters to the base image either - I think I struggled with that at some point

**Comment by ekkis** [comment]
hei, as a contributor, would you take a moment to look at my other issue? https://github.com/BlackGlory/caddy-proxy/issues/9 I could use someone's input
